source/sync/get-started/stream-data-from-client-to-atlas.txt

```diff
@@ -0,0 +1,267 @@
+.. _stream-data-from-client-to-atlas:
+
+===================================================
+Stream Data Unidirectionally from a Client to Atlas
+===================================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+If you have a mobile or client application that produces a large volume of 
+data you'd like to stream to MongoDB Atlas, you can sync data unidirectionally
+using Device Sync. We call the feature that enables this unidirectional sync
+**Asymmetric Sync**.
+
+You might want to sync data unidirectionally in IoT applications, such as 
+a weather sensor sending data to the cloud. Asymmetric sync is also useful 
+for writing other types of immutable data where you do not require conflict 
+resolution, such as creating invoices from a retail app or logging 
+application events.
+
+Asymmetric Sync is optimized to provide performance improvements for heavy 
+client-side *insert-only* workloads. You cannot read this data from the
+realm where you're streaming it.
+
+Follow these high-level steps to get started:
+
+Set up Atlas Access
+-------------------
+
+.. procedure::
+
+   .. step:: Create an Atlas App Services App
+
+      The App Services application is the gateway that enables your client 
+      device to connect to MongoDB Atlas. When you create an App, you name
+      it, link it to an Atlas cluster, and specify the deployment model and
+      deployment region that work best for your application.
+
+      You can create an App Services App via the :ref:`Atlas UI 
+      <create-app-ui>`, the :ref:`Atlas App Services Command Line Interface 
+      <create-app-cli>`, or the :ref:`App Services Admin API <create-app-api>`.
+
+   .. step:: Configure Device Sync and Asymmetric Sync
+
+      Device Sync is the service that manages synchronizing data to Atlas 
+      and across your client devices. Device Sync, together with the 
+      Realm SDKs, automatically manages network connectivity, user 
+      authentication, and user permissions and data access.
```

Comment:
User authentication -> not entirely true? The app services does that part.

Also not sure what you mean exactly by "data access".